# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1366872308 14400
#      Thu Apr 25 02:45:08 2013 -0400
# Node ID 68176102fe07d5a1061abbf5185aac25421cd712
# Parent  662a712b8fd570be07e92a660b74c09c28e04141
* load-path.cc (Faddpath): Doc fix.

diff --git a/libinterp/interpfcn/load-path.cc b/libinterp/interpfcn/load-path.cc
--- a/libinterp/interpfcn/load-path.cc
+++ b/libinterp/interpfcn/load-path.cc
@@ -2160,17 +2160,17 @@ No checks are made for duplicate element
 
   return retval;
 }
 
 DEFUN (addpath, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn  {Built-in Function} {} addpath (@var{dir1}, @dots{})\n\
 @deftypefnx {Built-in Function} {} addpath (@var{dir1}, @dots{}, @var{option})\n\
-Add @var{dir1}, @dots{} to the current function search path.  If\n\
+Add named directories to the function search path.  If\n\
 @var{option} is \"-begin\" or 0 (the default), prepend the\n\
 directory name to the current path.  If @var{option} is \"-end\"\n\
 or 1, append the directory name to the current path.\n\
 Directories added to the path must exist.\n\
 \n\
 In addition to accepting individual directory arguments, lists of\n\
 directory names separated by @code{pathsep} are also accepted.  For example:\n\
 \n\
